ui/src/App.tsx

```diff
@@ -77,13 +75,13 @@ interface FormEntryProps {
 }
 
 const App = () => {
-  const currentHour: number = new Date().getHours();
+  const datePlusHours: Function = (date: Date, hours: number): Date => {
+    date.setHours(date.getHours() + hours);
+    return date;
+  };
```

Comment:
I'll put this in `lib`.

Reply:
Let's make sure to unit test this when we move it to `lib` as well.

Reply:
Added here! e69df65
<img width="381" alt="image" src="https://user-images.githubusercontent.com/31298051/211723617-323af9ce-2b55-4c7a-afe4-c823f2948b01.png">